java/app/src/main/AndroidManifest.xml

```diff
@@ -10,20 +10,32 @@
         android:supportsRtl="true">
 
         <activity
-            android:name="CheckoutActivity"
+            android:name=".activity.CheckoutActivity"
             android:label="@string/store_name"
-            android:screenOrientation="portrait">
+            android:screenOrientation="portrait"
+            android:theme="@style/Theme.AppCompat">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
 
-        <!-- This element is required in order to use the PaymentsClient -->
+        <activity
+            android:name=".activity.PaymentTransparentActivity"
+            android:launchMode="singleTask"
+            android:taskAffinity=""
```

Comment:
Do you need to define a taskAfinity even if it is empty?

Reply:
Yep, an empty taskAffinity is used to explicitly reject any affinity to any existing task. There's somewhere in the Android docs where this is covered.